image/image_linux.go

```diff
@@ -797,7 +823,7 @@ var setupSeed = func(tsto *tooling.ToolingStore, model *asserts.Model, opts *Opt
 	// know the correct assertion max format to use.
 	// Fetch assertions tentatively into a temporary database
 	// and later either copy them or fetch more appropriate ones.
-	tmpDb := db.WithStackedBackstore(asserts.NewMemoryBackstore())
+	tmpDb := s.db.WithStackedBackstore(asserts.NewMemoryBackstore())
```

Comment:
nitpick: I'm not entirely a fan of these imageSeeder field accesses, before it was mostly s.f, now it's also s.db. nothing to do immediately but if we can think of an approach for later would be good, either we go back to start returning db and f, or we have more methods on imageSeeder I suppose

Reply:
It's a good point, I agree. I'll make a note of this in my TODO